jina/resources/executors._merge.yml

```
@@ -4,7 +4,11 @@ metas:
   name: merge
 requests:
   on:
-    [SearchRequest, TrainRequest, IndexRequest, DeleteRequest, UpdateRequest]:
-      - !ReduceDriver {}
+    [SearchRequest, TrainRequest, IndexRequest]:
+      - !ReduceAllDriver
+        with:
+          traversal_paths: ['r']
```

I think this should be 'c' by default
```suggestion
          traversal_paths: ['r']
```

---

using `r` makes it identical as `_merge_root`

---

That is the point